### PR TITLE
feat: add credits

### PR DESCRIPTION
**Pull request by DanielHabenicht**
Add page for displaying people that have worked on this app.

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
image

File: apps/official/components/pages/Credits.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            {
+              contributers.contributors.map(el => <li><a href="{el.profile}" target='_blank' rel="noopener">{el.name}</a></li>)
+            }
+            </ul>
+          </Typography>
+        </section>
+        <section>
+          <Typography variant="h2">Unternehmen</Typography>
+          <Typography variant="body1">
+            Der Firma Portbluesky für die Freistellung von ... .
```
Comment (DanielHabenicht) [inline apps/official/components/pages/Credits.tsx:29]:
```suggestion
            Port Blue Sky für die technische Umsetzung der App durch Ihre Mitarbeiter sowie das Organisieren von weiterer Unterstützung durch Voluntäre aus der Node.js Community.
```

File: apps/official/index.ts [hunk 2]
```diff
@@ -71,6 +72,12 @@ export const config: AppConfig = {
         route: "/privacy-statement",
         Component: Privacy,
       },
+      {
+        id: "credits",
+        title: "Über die CovApp",
```
Comment (alexanderthieme) [inline apps/official/index.ts:77]:
```suggestion
        title: "Über die CovMap",
```

**Pull request merged**